packages/shared/lib/core/nfts/stores/selected-account-nfts.store.ts

```diff
@@ -1,5 +1,7 @@
-import { selectedAccount } from '@core/account'
 import { derived, Readable } from 'svelte/store'
+
+import { selectedAccount } from '@core/account/stores'
+
```

Comment:
any reasons for the empty lines?

Reply:
I think Matt added this, since the import comes from another module